eva/catalog/services/df_service.py

```diff
@@ -93,23 +93,39 @@ def dataset_object_by_name(self, database_name, dataset_name,
             DataFrameMetadata - metadata for given dataset_name
         """
         return self.model.query.filter(
-            self.model._name == dataset_name).one_or_none()
+            self.model._name == dataset_name
+        ).one_or_none()
 
-    def drop_dataset_by_name(self,
-                             database_name: str,
-                             dataset_name: str) -> bool:
-        """ Delete dataset from the db
-            Arguments:
-                database_name  (str): Database to which dataset belongs
-                dataset_name (str): name of the dataset
-            Returns:
-                True if successfully removed else false
+    def drop_dataset_by_name(self, database_name: str, dataset_name: str):
+        """Delete dataset from the db
+        Arguments:
+            database_name  (str): Database to which dataset belongs
+            dataset_name (str): name of the dataset
+        Returns:
+            True if successfully removed else false
         """
         try:
             dataset = self.dataset_object_by_name(database_name, dataset_name)
             dataset.delete()
-        except Exception:
-            logger.error(
-                "Delete dataset failed for name {}".format(dataset_name))
-            return False
-        return True
+        except Exception as e:
+            err_msg = "Delete dataset failed for name {} with error {}".format(
+                dataset_name, str(e)
+            )
+            logger.error(err_msg)
```

Comment:
Shall we use `logger.exception`? So it prints the full stack trace in the server log.